src/act.drive.cpp

```diff
@@ -888,6 +888,11 @@ ACMD(do_control)
         if (GET_OBJ_TYPE(GET_EQ(ch, x)) == ITEM_RCDECK)
           has_rig = TRUE;
 
+  if (!has_rig)
+    for (cyber = ch->cyberware; cyber; cyber = cyber->next_content)
+      if (GET_OBJ_VAL(cyber, 0) == CYB_CRD))
```

Comment:
`GET_CYBERWARE_TYPE` instead of the raw obj_val call, please!

Reply:
Is the `(cyber, 0)` value reference still needed to reference type?

Reply:
I'm assuming not, since utils.h has a convenience definition